lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -5201,13 +5299,16 @@ ParseResult FIRCircuitParser::parseLayer(CircuitOp circuit) {
     consumeToken();
 
     hw::OutputFileAttr outputDir;
-    if (getToken().getKind() == FIRToken::string) {
-      auto text = getToken().getStringValue();
-      if (text.empty())
-        return emitError() << "output directory must not be blank";
-      outputDir = hw::OutputFileAttr::getAsDirectory(getContext(), text);
-      consumeToken(FIRToken::string);
+    if (consumeIf(FIRToken::comma)) {
```

Comment:
I don't think we want a trailing comma here.  `,:` shouldn't be valid I think.

Reply:
I backed out the trailing commas change.